packages/mongodb-chatbot-eval/README.md

```diff
@@ -1,3 +1,3 @@
 # MongoDB Chatbot Evaluation CLI
 
-TODO
+CLI application for evaluating MongoDB Chatbot Framework quality.
```

Comment:
i'd rather build out README in a future ticket where the full set of functionality is ready. also, rather than explain in detail, i'd rather just link to the docs site and let the details live there to avoid duplication. 

made clear in the docs ticket for the epic that it should include README updates - https://jira.mongodb.org/browse/EAI-233